src/porepy/numerics/pde_solver.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
+import scipy.sparse as sps
+
+from porepy.grids import structured
+from porepy.grids.grid_bucket import GridBucket
+from porepy.params.data import Parameters
+from porepy.params import tensor
+from porepy.params import bc
+from porepy.numerics.mixed_dim.solver import Solver
+
+
+class AbstractSolver(object):
```

Comment:
This, and its children, are time stepping schemes, whereas Parabolic and Darcy are mainly concerned with spatial discretization. How do we combine the two? Is it possible?

Reply:
AbstractSolver takes a Parabolic as an input in the initialization

Reply:
Perfect.